docs/ui/components/Markdown/Blockquote.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { render } from '@testing-library/react';
+
+import { Blockquote } from './Blockquote';
+
+import { P } from '~/ui/components/Text';
+
+describe('Blockquote', () => {
+  it('renders with default info icon', () => {
+    const { getByTitle } = render(<Blockquote>Hello</Blockquote>);
+    expect(getByTitle('Info-icon')).toBeInTheDocument();
+  });
+
+  it('renders with emoji as icon', () => {
+    const { getByText } = render(<Blockquote>🎨 Hello</Blockquote>);
+    expect(getByText('🎨')).toBeInTheDocument();
+    expect(getByText('🎨')).not.toBe(getByText('Hello'));
+  });
+
+  it('renders with type from special warning emoji ⚠️', () => {
+    const { getByTitle } = render(<Blockquote>⚠️ Careful</Blockquote>);
+    expect(getByTitle('Warning-icon')).toBeInTheDocument();
+  });
+
+  it('renders with type from special error emoji ❌', () => {
+    const { getByTitle } = render(<Blockquote>❌ Watch out</Blockquote>);
+    expect(getByTitle('Error-icon')).toBeInTheDocument();
+  });
+
+  it('renders with multiple emojis', () => {
+    const { getByText, getByTitle } = render(<Blockquote>⚠️ Watch ❌ out</Blockquote>);
+    expect(getByTitle('Warning-icon')).toBeInTheDocument();
+    expect(getByText('Watch ❌ out')).toBeInTheDocument();
+  });
+
+  // Markdown adds unnecessary paragraphs inside blockquotes.
+  // Test if the blockquote extracts the first emoji with paragraphs too.
```

Comment:
In hindsight, they are actually correct in adding the `P` tag. If a blockquote is written like:

```
> 🎨 **Look how cool that is!***
>
> Bla bla, it's because it's Expo, bla bla
```

We want to render these two separate blocks with a margin, the `P` tag takes care of that. So I think they are right, but it's just super annoying to also have to support it 🙈  (luckily we do now)

```suggestion
  // Markdown adds paragraphs inside blockquotes, which is useful for multiline blockquotes.
  // Test if the blockquote extracts the first emoji with paragraphs too.
```